applications/webhooks/src/webhooks_shared_listener.erl

```diff
@@ -327,3 +332,15 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 %%% Internal functions
 %%%===================================================================
+
+-spec should_handle_federated_events(binaries()) -> kz_proplist().
+should_handle_federated_events([]) -> [];
+should_handle_federated_events([H | T]) ->
+    case kz_util:to_binary(kz_nodes:local_zone()) == H of
+        'true' ->
+            [{<<"should_handle_federated">>, 'true'}];
+        _ when T == [] ->
+            [{<<"should_handle_federated">>, 'true'}];
```

Comment:
What if webhooks is only running just in one zone? I'm checking here if this is the only zone that has webhooks running we should process the event.